docs/_docs/reference/experimental/anonymous-context-bounds.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: doc-page
+title: "Anonymous Context Bounds"
+---
+
+This page describes experimental support for anonymous context bounds Scala 3, available in 3.2 or later. It is enabled by the language import
+```scala
+import language.experimental.anonymousContextBounds
+```
+The reason for publishing this extension now is to get feedback on its usefulness.
+
+## Why Anonymous Context Bounds?
+
+Typeclasses are [first-class consideration of Scala 3](https://docs.scala-lang.org/scala3/book/ca-type-classes.html) 
+and [context bound sugar](https://docs.scala-lang.org/scala3/book/ca-context-bounds.html) already exists to support their use.
+Taking an example from [here](https://docs.scala-lang.org/scala3/book/ca-type-classes.html), context bounds allow one
+to write
+```scala
+def showAll[S: Showable](xs: Set[S]): Unit = xs.foreach(x => println(x.show))
```

Comment:
for clarity, you should show the definition of `Showable` first - but I made a suggestion lower in the file that perhaps `Ordering` is fine enough and can link to the definition

Reply:
Done